### ISSUE TITLE
ghcr.io/basecamp/kamal:latest points to v1.9.1 : unable to use Kamal 2 with the docker command

### ISSUE THREAD

**Issue by naofumi** [basecamp/kamal#1029]
As shown in the link and screenshot below, currently `ghcr.io/basecamp/kamal:latest` points to v1.9.1

As a result, when I try to use Kamal using Docker and the command below, I get v1.9.1

```sh
alias kamal='docker run -it --rm -v "${PWD}:/workdir" -v "/run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock" -e SSH_AUTH_SOCK="/run/host-services/ssh-auth.sock" -v /var/run/docker.sock:/var/run/docker.sock ghcr.io/basecamp/kamal:latest'
```

https://github.com/basecamp/kamal/pkgs/container/kamal

<img width="761" alt="image" src="https://github.com/user-attachments/assets/7c99ec17-a9c0-45ae-905d-f667c6620972">

**Comment by djmb** [comment]
https://github.com/basecamp/kamal/pull/1030 should prevent this in future.

**Comment by djmb** [comment]
I've released v2.1.0 which has reclaimed the `latest` tag and merged #1030 to prevent this happening in future 👍
